continuous-integration/linux/test.sh

```diff
@@ -2,14 +2,8 @@
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 ROOT_DIR=$(realpath "$SCRIPT_DIR/../..")
-VENV=$(mktemp --tmpdir --directory zivid-python-test-venv-XXXX) || exit $?
 
-python -m venv $VENV || exit $?
-source $VENV/bin/activate || exit $?
-
-pip install --upgrade pip || exit $?
-pip install "$ROOT_DIR" || exit $?
-pip install -r "$SCRIPT_DIR/../requirements-build-and-test.txt" || exit $?
+pip install --requirement "$SCRIPT_DIR/../python-requirements/test.txt" || exit $?
```

Comment:
Why did you remove the virtual environment?

Reply:
Now you will have access to packages installed by earlier steps.

Reply:
I decided to remove the virtual environments because I do not think they provide much value as of now. We could move the virtual creation to the build.sh step, but I decided against it, mostly because it will be difficult to reason about the code in test.sh (just by looking at that file it is not clear that the python/pip we use, which might be confusing).